uint32 requestID
string customerID
string status
string pickUpLocation
string dropOffLocation